src/io/flutter/run/daemon/DeviceDaemon.java

```diff
@@ -262,6 +263,11 @@ public void onDaemonLogMessage(@NotNull DaemonEvent.LogMessage message) {
       LOG.info("flutter device watcher: " + message.message);
     }
 
+    @Override
+    public void onDaemonShowMessage(@NotNull DaemonEvent.ShowMessage event) {
+      FlutterMessages.showMessage(event.title, event.message);
```

Comment:
We're driving the UI directly from flutter_tools and I worry that it's not obvious to someone working on the flutter_tools side what the UI effect will be. So if that's how we want to do it, maybe the message should be called daemon.showToast?

Alternately, if we want to abstract away from the UI in the protocol, there should probably be a unique id for each kind of message, and a switch here to decide how to display each kind of message.